src/lang-go.ts

```diff
@@ -660,6 +662,11 @@ export async function activateUsingWebSockets(ctx: sourcegraph.ExtensionContext)
             },
         })
     )
+    const panelView = sourcegraph.app.createPanelView(IMPL_ID)
+    panelView.title = 'Go ifaces/impls'
```

Comment:
The language name "Go" doesn't show up on the other tabs. Any reason it's on this one?

Would a title like "Implementations" be accurate (and allow you to avoid the abbreviation)?

I'm still seeing an IMPLEMENTATION tab:

![image](https://user-images.githubusercontent.com/1387653/54330403-c873ed00-45d2-11e9-8dab-cb2eb5829e6d.png)

Is that from an older version of sourcegraph/sourcegraph compared to your screenshot?

Reply:
> The language name "Go" doesn't show up on the other tabs. Any reason it's on this one?

When I demo Go implementations, frequently people don't know what that would be, because looking up Go implementations is rare. People think "oh, is that the implementation of the function?" So I thought this name would best communicate it. And also a slight nudge to communicate that these panel titles are extensible. 

(Constrast with TypeScript, where the concept of "Implementations" is much more obvious what it is. Nobody has wondered what it means for TypeScript.)

> Is that from an older version of sourcegraph/sourcegraph compared to your screenshot?

Yes, this is fixed (removed) in https://github.com/sourcegraph/sourcegraph/pull/1697